parsers/jscript.c

```diff
@@ -2005,6 +2007,9 @@ static bool parseES6Class(tokenInfo *const token,tokenInfo * const parent)
 		if(isType(token,TOKEN_CLOSE_CURLY))
 			break;
 		
+		if(isKeyword(token,KEYWORD_static))
+			readToken(token);
+		
```

Comment:
Sure, we could add properties.
I'm still a bit afraid of messing up too much with this parser. Need to understand it better first.